### DISCUSSION TITLE
Haven't received an update on the Campus Experts Application.

### DISCUSSION THREAD

**Discussion by nityaa3009** [community/community#139106] [GitHub Education]
It's been more than 2 weeks since I've submitted my application for the Github Campus Experts Program. Does this mean that I've been rejected? Or when can I expect mail?

**Comment by hyper-ayoub** [comment]
Hi bro so please wait the process it will take more time but i recommend you to submit all of your information to the support because the max-time that will take i think is 24 day   so just wait until you get it in email  ✔✔@nityaa3009

**Reply by nityaa3009** [reply]
hey, thanks a lot.
but it's the 24th now and still no update :(

**Comment by Injamul2k24** [comment]
it may be some problem in your application .

**Comment by vraj00222** [comment]
today it was supposed to be out the result of first round but no update or email anyone got any response
